toolkit/tools/pkg/imagecustomizerlib/customizeboot.go

```diff
@@ -9,12 +9,20 @@ import (
 	"path/filepath"
 	"regexp"
 
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/imagecustomizerapi"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/imagegen/installutils"
 	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
 	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/safechroot"
 )
 
 var (
 	linuxCommandLineRegex = regexp.MustCompile(`\tlinux .* (\$kernelopts)`)
+
+	linuxCommandLineSELinuxRegex = regexp.MustCompile(`\tlinux .* (security=selinux selinux=1( enforcing=1)?) `)
```

Comment:
To be on the safe side, what do you think about this small modification:
```suggestion
	linuxCommandLineSELinuxRegex = regexp.MustCompile(`\tlinux .* (security=selinux\s+selinux=1(\s+enforcing=1)?) `)
```

Reply:
Also, I see there's a space at the end of the regex. I'm not an expert here but I want to double-check: is it guaranteed that space is going to be there (the arguments won't end yet)?

Reply:
If the image was created by the imager and wasn't manually modified by the user, then the end space is guaranteed to be there. Adding the end space to the regex means I don't need to worry about adding extra spaces when inserting the new args.

Reply:
`\s` includes newlines, which I am not keen to include in the regex.

Reply:
Got it. In that case could we add a comment explaining why we guarantee the spaces?

And you're right about `\s`. I think `[[:blank:]]` might work better then but I have no strong need to modify this regex.